entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,198 @@
+        range: '/common/images/range.png',
+        vegetables_range_white: '/common/images/vegetables_range_white.png',
+        green_range: '/common/images/green_range.png',
+        orange_range: '/common/images/orange_range.png',
+        violet_range: '/common/images/violet_range.png',
+        microphone: '/common/images/microphone.png',
+        clock: '/common/images/clock.png',
+        favourite_active: '/common/images/favourite_active.png',
+        favourite_inactive: '/common/images/favourite_inactive.png',
+        home_icon: '/common/images/home.png',
+        storage_icon: '/common/images/storage.png',
+        music_icon: '/common/images/music.png',
+        settings_icon: '/common/images/settings.png',
+        view_icon: '/common/images/view_icon.png',
+        add_icon: '/common/images/add_icon.png',
+        pink_button: '/common/images/pink_button.png',
+
+        //      T E X T
+        storage_settings:'Storage Settings',
+        dinner_text:'Dinner Recepies for you',
+        all_recp_text:'All Recepies',
+        food_remain: 'Food Remaining',
+        home_txt: "Home",
+        storage_txt: "Storage",
+        music_txt: "Music",
+        settings_txt: "Settings",
+        audio_assist1:"How can I help you?",
+        audio_assist2:"Tap to Speak",
+        your_storage:"Your Storage",
+        view_text:"View",
+        add_text:"Add",
+        door_text:"Refrigerator Doors",
+        time:"",
+        day_date:"",
+
+        //      Device Details
+        username: 'UserName',
+        location_city:"Bangalore",
+        //day_date:"Wednesday, 17 Dec",
+        temp_red:"30°",
+
+        //      Music Information
+        music_cover_image:'/common/images/dreams_img.png',
+        song_name: 'Dreams',
+        song_genre: 'Romantic',
+
+        //      Food Remaining tab
+        vegetable_percentage:"72%",
+        fruits_percentage:"50%",
+        meat_percentage:"20%",
+
+        //      Storage Setting Items
+        st_item_1: '/common/images/chicken.png',
+        st_item_1_name:"Fresh Chicken",
+        st_item_1_weight:"1kg",
+        st_item_2: '/common/images/vegetables.png',
+        st_item_2_name:"Vegetables",
+        st_item_2_weight:"3kg",
+        st_item_3: '/common/images/spinach.png',
+        st_item_3_name:"Spinach",
+        st_item_3_weight:"500g",
+        st_item_4: '/common/images/fruits.png',
+        st_item_4_name:"Fruits",
+        st_item_4_weight:"3kg",
+        st_item_5: '/common/images/fruits.png',
+        st_item_5_name:"Milk",
+        st_item_5_weight:"2lts",
+
+        //      Refrigeator Settings
+        door_status:"Locked",
+        door_status_toggle:"Unlock",
+        superfreeze_toggle_txt:"On",
+        superfreeze_toggle_txt_bool:"",
+        bottom_box_left_toggle_text: "Freeze",
+        bottom_box_right_toggle_text: "Fresh",
+        crisp_fresh_toggle_txt:"On",
+        crisp_fresh_toggle_txt_bool:"",
+
+
+        //      Recepie 1 Information
```

Comment:
Please update few more data as well based on apis;
For example for Recipe information, 
1. maintain an array of not pre-defined size and with a structure (for example holds objects which can maintain image path, name etc ) in data.
2. Maintain the hardcoded recipe data in a JSON file with around 4 to 5 entries
3. Define a function called getRecipeList -> This function reads the JSON file content and update the data to the array mentioned in point 1
4. Call getRecipeList method in onInit 
5. Also call getRecipeList if in case on click of any vie if this list has to be updated (if any such requirement)
6. Now at UI, loop through the array and display the data as per the content in the array
7.  Also handle the case where the recipe array is empty (No recipes), design can be discussed with designer

Similar way handle for other possible apis.